volcreate

```diff
@@ -432,18 +432,18 @@ $0 =~ s%.*/%%;
 # ACLs, in order to allow things like -clear.
 Getopt::Long::config('bundling', 'no_ignore_case');
 GetOptions(
+    'P|proportion=f'       => \$VOLCREATE_CUTOFF,
+    'Q|quiet'              => \$quiet,
+    'V|version'            => \$version,
```

Comment:
using uppercase for quiet and version is inconsistent with all of the other tools in the afs-admin-tools suite.  

is a single letter option required for proportional?